01-node-tutorial/answers/02-globals.js

```diff
@@ -0,0 +1,8 @@
+console.log(__dirname)
+console.log(__filename)
+process.env.MY_VAR = "Hi there!";
```

Comment:
You correctly identified how to access the system environment variables, however, for the assignment you need to retrieve the variable, not set it. If you are having trouble with your terminal/console/command line, DM me and I can help you out!